### DISCUSSION TITLE
90FPS Option

### DISCUSSION THREAD

**Discussion by MadEngineer75** [open-goal/jak-project#3223] [General]
The newly released Steam Deck OLED has a 90hz screen. So, a 90FPS option would allow users to take full advantage of it. 

Thank you and I appreciate the work everyone is doing.

**Comment by xTVaser** [comment]
This isn't possible in jak 1, the framerate has to be a multiple of 300 as the code currently stands.  It would be a significant amount of work to decouple this.

This should be possible in jak 2/3 however.

Are you saying that on the steam deck 90hz screen, there isn't the 100fps option?

**Reply by MadEngineer75** [reply]
The 100fps option is available. However, if V-Sync is turned on, the game gets limited to 90fps and slow down occurs. I hate tearing, so I would never play with V-Sync off, but I decided to try it out of curiosity. If I disable the framerate limit and turn on allow tearing, the game reached 100fps and plays at a normal speed. However, the frametimes are inconsistent and produce notable judder. Curiously, I don't see any screen tearing, so I believe some sort of v-sync is still active and causes the judder despite me turning off the in-game and SteamOS v-sync.

**Reply by Psychosikh** [reply]
Same slowdown occurs in Jak 2, a 90 fps option would appreciated

**Comment by Hat-Kid** [comment]
#3245 will add a 90 FPS option in Jak 2.
